corehq/apps/accounting/management/commands/suppress_duplicate_invoices.py

```diff
@@ -0,0 +1,97 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+from corehq.apps.accounting.interface import get_subtotal_and_deduction
+from corehq.apps.accounting.models import Invoice, CreditLine, FeatureType
+from corehq.util.dates import get_first_last_days
+
+
+class Command(BaseCommand):
+    help = 'Put credit back for duplicate invoices and suppress them'
+
+    def add_arguments(self, parser):
+        parser.add_argument('year', type=int, help='The year of the statement period.')
+        parser.add_argument('month', type=int, help='The month of the statement period.')
+        parser.add_argument('note', type=str,
+                            help='A note to be added when crediting back the amount')
+        parser.add_argument('--dry-run', action='store_true', default=False)
+
+    def handle(self, *args, **kwargs):
+        year = kwargs['year']
+        month = kwargs['month']
+        note = kwargs['note']
+        dry_run = kwargs['dry_run']
+        dry_run_tag = '[DRY_RUN]' if dry_run else ''
```

Comment:
```suggestion
        dry_run_tag = '[DRY RUN] ' if dry_run else ''
```
space for readability considering how it's being used below

Reply:
Add spaces for both `dry_run_tag` here: d4dd616d3399a1650af84d4a51670fa0e64a4031